recipes/exprtk/all/conanfile.py

```diff
@@ -35,16 +35,9 @@ def validate(self):
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
-    def _extract_license(self):
-        exprtk_header_file = "exprtk.hpp"
-        file = os.path.join(self.source_folder, exprtk_header_file)
-        file_content = load(self, file)
-        license_end = "/MIT                        *"
-        license_contents = file_content[2:file_content.find(license_end) + len(license_end)]
-        save(self, os.path.join(self.package_folder, "licenses", "LICENSE"), license_contents)
-
     def package(self):
-        self._extract_license()
```

Comment:
This is still needed for the older versions

Reply:
oh yeah you're right! should I place some kind of conditional statement based on the tag?

Reply:
Yes, you can do `from conan.tools.scm import Version` and then do something like
```python
if Version(self.version) < "0.3.0":
    # do stuff
```